$NetBSD: patch-ad,v 1.7 2011/05/21 15:32:59 wiz Exp $

1. chunk: adapt installation paths for pkgsrc; make UDEV optional
2. chunk: adapt installation path for pkgsrc
3. chunk: adapt path for NetBSD
4. chunk: link against libossaudio where it exists (variable supplied
by pkgsrc)

--- src/Makefile.am.orig	2009-11-22 19:57:04.000000000 -0800
+++ src/Makefile.am	2010-10-01 10:11:05.000000000 -0700
@@ -26,13 +26,17 @@
 
 pulseincludedir=$(includedir)/pulse
 pulsecoreincludedir=$(includedir)/pulsecore
-pulseconfdir=$(sysconfdir)/pulse
+pulseconfdir=$(EGDIR)
 pulselibexecdir=$(libexecdir)/pulse
-xdgautostartdir=$(sysconfdir)/xdg/autostart
+xdgautostartdir=$(PREFIX)/share/gnome/autostart
 alsaprofilesetsdir=$(datadir)/pulseaudio/alsa-mixer/profile-sets
 alsapathsdir=$(datadir)/pulseaudio/alsa-mixer/paths
+if HAVE_UDEV
 udevrulesdir=/lib/udev/rules.d
-dbuspolicydir=$(sysconfdir)/dbus-1/system.d
+else
+udevrulesdir=
+endif
+dbuspolicydir=$(EGDIR)
 
 ###################################
 #            Defines              #
@@ -42,7 +46,7 @@
 if OS_IS_WIN32
 PA_DEFAULT_CONFIG_DIR=%PULSE_ROOT%
 else
-PA_DEFAULT_CONFIG_DIR=$(pulseconfdir)
+PA_DEFAULT_CONFIG_DIR=$(sysconfdir)/pulse
 endif
 
 ###################################
@@ -83,7 +87,7 @@
 	-DPA_SYSTEM_STATE_PATH=\"$(PA_SYSTEM_STATE_PATH)\" \
 	-DAO_REQUIRE_CAS \
 	-DPULSE_LOCALEDIR=\"$(pulselocaledir)\" \
-	-DPA_MACHINE_ID=\"$(localstatedir)/lib/dbus/machine-id\" \
+	-DPA_MACHINE_ID=\"$(localstatedir)/db/dbus/machine-id\" \
         -DPA_ALSA_PATHS_DIR=\"$(alsapathsdir)\" \
         -DPA_ALSA_PROFILE_SETS_DIR=\"$(alsaprofilesetsdir)\"
 
@@ -1419,7 +1423,7 @@
 # OSS
 
 liboss_util_la_SOURCES = modules/oss/oss-util.c modules/oss/oss-util.h
-liboss_util_la_LDFLAGS = -avoid-version
+liboss_util_la_LDFLAGS = -avoid-version $(LIBOSSAUDIO)
 liboss_util_la_LIBADD = libpulsecore-@PA_MAJORMINORMICRO@.la libpulsecommon-@PA_MAJORMINORMICRO@.la libpulse.la
 
 module_oss_la_SOURCES = modules/oss/module-oss.c
